### PR TITLE
BookList component

### PR DESCRIPTION
**Pull request by EmmanuelWLX**
### Summary
- Added a BookList component to show all the Book components rendered from the json file
- The Book component is now an internal component of BookList

### Screenshots
screencast-Genymotion-2019-05-30_15 36 08 765_765

### Trello Card
https://trello.com/c/3bTuSeWb/9-n-53-lista-de-libros

### CODE REVIEW

File: LibraryApp/src/app/components/BookList/components/Book/styles.js [hunk 1]
```diff
@@ -1,9 +1,12 @@
 import { StyleSheet } from 'react-native';
-import { FONT_SIZE_LARGE, FONT_SIZE_MEDIUM } from '../../../constants/dimensions';
-import { white } from '../../../constants/colors';
+import { FONT_SIZE_LARGE, FONT_SIZE_MEDIUM } from '../../../../../constants/dimensions';
```
Comment (mmamaniwolox) [inline LibraryApp/src/app/components/BookList/components/Book/styles.js]: Relative imports, again :P

File: LibraryApp/src/app/index.js [hunk 2]
```diff
@@ -1,16 +1,16 @@
 import React, { Component } from 'react';
 import { View } from 'react-native';
 
-import Book from './components/Book';
-import * as books from '../books.json';
+import BookList from './components/BookList';
 import styles from './styles';
 
-class App extends Component {
+const booksTest = require('../books.json');
```
Comment (mmamaniwolox) [inline LibraryApp/src/app/index.js]: Why dont you use import?

File: LibraryApp/.eslintrc.js [hunk 3]
```diff
@@ -2,15 +2,6 @@ module.exports = {
...
-    "import/resolver": {
-      "babel-module": {}
-    }
-  },
-  plugins: [
-    "module-resolver"
-  ]
+    'import/no-extraneous-dependencies': 'off'
+  }
 };
```
Comment (mmamaniwolox) [inline LibraryApp/.eslintrc.js]: tailing

File: LibraryApp/.vscode/settings.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
+    "files.trimTrailingWhitespace": true,
+    "editor.detectIndentation": false,
+    "editor.tabSize": 2
+}
```
Comment (mmamaniwolox) [inline LibraryApp/.vscode/settings.json]: I would add `.vscode` to gitignore

File: LibraryApp/jsconfig.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "baseUrl": "./src",
+    "baseUrl": ".",
```
Comment (mmamaniwolox) [inline LibraryApp/jsconfig.json]: why do you keep using this file? we should use babel for relative routes

File: LibraryApp/src/app/components/BookList/components/Book/styles.js [hunk 6]
```diff
@@ -2,7 +2,11 @@ import { StyleSheet } from 'react-native';
 import { white } from '@constants/colors';
 import { FONT_SIZE_LARGE, FONT_SIZE_MEDIUM } from '@constants/dimensions';
 
-const IMAGE_SIZE = 80;
+const IMAGE_HEIGHT = 80;
+// eslint-disable-next-line no-magic-numbers
```
Comment (mmamaniwolox) [inline LibraryApp/src/app/components/BookList/components/Book/styles.js]: lol, what?

Comment (EmmanuelWLX) [reply LibraryApp/src/app/components/BookList/components/Book/styles.js]: my bad 😓

Comment (EmmanuelWLX) [reply LibraryApp/src/app/components/BookList/components/Book/styles.js]: i swear i deleted it in a newer pr 🤞

File: LibraryApp/src/app/components/BookList/index.js [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+class BookList extends Component {
+  keyExtractor = ({ id }) => id;
+
+  renderItem = ({ item }) => <Book title={item.title} author={item.author} photo={item.image_url} />;
+
+  render() {
+    const { data } = this.props;
+    return (
+      <View style={styles.container}>
```
Comment (mmamaniwolox) [inline LibraryApp/src/app/components/BookList/index.js]: Do you need this?

Comment (EmmanuelWLX) [reply LibraryApp/src/app/components/BookList/index.js]: i love views (? but really i don't need it 👍

**Pull request merged**